tests/ci_tests_basic.yml

```diff
@@ -0,0 +1,83 @@
+---
+- name: empty play to gather facts
+  hosts: all
+- import_playbook: playbooks/rhel7_deps.yml
+  when:
+    - ansible_distribution == "RedHat"
+    - ansible_distribution_major_version == '7'
+- import_playbook: playbooks/centos7_deps.yml
+  when:
+    - ansible_distribution == "CentOS"
+    - ansible_distribution_major_version == '7'
+- name: Run tox tests and nmstatectl show
+  hosts: all
+  vars:
+    - nmstatedir: /tmp/nmstate-git-checkout
+  tasks:
+    # Does not work when specifying multiple hosts:
+    # https://github.com/ansible/ansible/issues/27973
+    # do not run on CentOS/RHEL 6
+    #- meta: end_play
```

Comment:
Could you either add a TODO or remove the code? I personally don't like commented-out code.

Reply:
done